docs/sql/reference/show_tables.txt

```diff
@@ -0,0 +1,42 @@
+.. highlight:: psql
+.. _ref-show-tables:
+
+===========
+SHOW TABLES
+===========
+
+Lists the tables in the database.
+
+Synopsis
+========
+
+::
+
+    SHOW TABLES [{FROM | IN} schema_name] [LIKE 'pattern']
```

Comment:
not in sync with parser: ``WHERE expression`` is missing here or not deleted in parser, if should not be supported.